[2025-10-02T23:38:13.385+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-02T23:38:13.434+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_taskk.task_numss scheduled__2025-10-02T23:37:00+00:00 [queued]>
[2025-10-02T23:38:13.449+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_taskk.task_numss scheduled__2025-10-02T23:37:00+00:00 [queued]>
[2025-10-02T23:38:13.453+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-02T23:38:13.483+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_numss> on 2025-10-02 23:37:00+00:00
[2025-10-02T23:38:13.497+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=508) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-02T23:38:13.501+0000] {standard_task_runner.py:63} INFO - Started process 510 to run task
[2025-10-02T23:38:13.501+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'first_taskk', 'task_numss', 'scheduled__2025-10-02T23:37:00+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmppxi63_vh']
[2025-10-02T23:38:13.507+0000] {standard_task_runner.py:91} INFO - Job 128: Subtask task_numss
[2025-10-02T23:38:13.532+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-02T23:38:13.628+0000] {task_command.py:426} INFO - Running <TaskInstance: first_taskk.task_numss scheduled__2025-10-02T23:37:00+00:00 [running]> on host 452c0657f430
[2025-10-02T23:38:13.850+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_taskk' AIRFLOW_CTX_TASK_ID='task_numss' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T23:37:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-02T23:37:00+00:00'
[2025-10-02T23:38:13.856+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-02T23:38:13.858+0000] {logging_mixin.py:188} INFO - random num is saved
[2025-10-02T23:38:13.859+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-02T23:38:13.861+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-02T23:38:13.884+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=first_taskk, task_id=task_numss, run_id=scheduled__2025-10-02T23:37:00+00:00, execution_date=20251002T233700, start_date=20251002T233813, end_date=20251002T233813
[2025-10-02T23:38:13.960+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-02T23:38:13.993+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-02T23:38:13.998+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
